packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -615,18 +618,94 @@ export const RemoteServerFactory = {
         }
     },
 
-    addSSRRenderer(app) {
-        // See - https://www.npmjs.com/package/webpack-hot-server-middleware#usage
-        const {buildDir} = app.options
-        const _require = eval('require')
-        const serverRenderer = _require(path.join(buildDir, 'server-renderer.js')).default
-        const stats = _require(path.join(buildDir, 'loadable-stats.json'))
+    addStaticAssetServing() {
+        // Handled by the CDN on remote
+    },
+
+    addDevServerGc() {
+        // This is a hook for the dev-server. The remote-server
+        // does GC in a way that is awkward to extract. See _createHandler.
+    },
+
+    /**
+     * Serve the /worker.js file.
+     *
+     * Unlike other bundle assets, the worker cannot be served from a
+     * bundle path. It must be served at the root of the site, so
+     * that it can control all pages and requests. This presents some
+     * challenges in caching. If we set a long age in the cache-control
+     * header, then CloudFront and/or browsers may continue to use
+     * outdated worker code. If we set a short value, we have to serve
+     * more requests. We cannot solve this by returning a redirect to the
+     * location of the worker file under the bundle path, because redirects
+     * are not valid responses to the request to load a service worker
+     * file for registration.
+     *
+     * The solution is to set a long value for s-maxage (CDN caching), plus
+     * a strong etag (to allow CDN-only revalidation), and to set maxage
+     * to 0 to prevent browser caching.
+     *
+     * See https://developers.google.com/web/updates/2018/06/fresher-sw
```

Comment:
The site has been directed to https://developer.chrome.com/docs/android/trusted-web-activity/, which does not seem relevant to us